build/integration-test.sh

```diff
@@ -25,7 +25,7 @@ TEST_TIMEOUT="30m"
 # Set default options
 TEST_OPTIONS="-tags=integration -timeout ${TEST_TIMEOUT} -check.suitep ${DOP}"
 # Regex to match apps to run in short mode
-SHORT_APPS="^PostgreSQL$|^MySQL$|Elasticsearch|^MongoDB$|Maria|^MSSQL$"
```

Comment:
We don't need to disable the app if image upgrade fixes the CI, right?

Reply:
I thought we build the image after merging and that the CI would fail if we don't remove it but seems like it passed anyway. I'll add it back and check